hw/ip/otbn/dv/model/otbn_trace_checker.cc

```diff
@@ -246,10 +246,12 @@ bool OtbnTraceChecker::MatchPair() {
     return false;
   }
 
-  // We've got a matching pair. Move the ISS data out of the (now defunct)
+  // We've got a matching insn pair. Move the ISS data out of the (now defunct)
```

Comment:
Ooh, probably "insn" is wrong here (because of the secure wipe U/V trace entries). Maybe drop this, or change to something like "matching pair of entries"?